pkgs/data/fonts/inconsolata/default.nix

```diff
@@ -1,20 +1,28 @@
-{ stdenv, fetchzip }:
+{ stdenv, fetchFromGitHub }:
 
-let
-  version = "1.010";
-in fetchzip {
+stdenv.mkDerivation rec {
   name = "inconsolata-${version}";
+  version = "2.001";
 
-  url = "http://www.levien.com/type/myfonts/Inconsolata.otf";
+  src = fetchFromGitHub {
+    owner = "google";
+    repo = "fonts";
+    rev = "4c3e95c802f8f12b78869ff50d552014de63f9c1";
+    sha256 = "1ndmsf4c0k36dakmps0vr7hhg5ss8m7ywja7v55xdrinvli58v2f";
+  };
```

Comment:
We have a package called google-fonts. Maybe we can take the source from there?

```
inherit (google) src version;
```

Reply:
Oh, I missed it. I think that it's nice to have the individual package regarding backward compatibility and package size. (`google-fonts` is bigger than the individual package.) If it's complicated to maintain an individual package, there is another method that split `google-fonts` into multiple packages (e.g., google-fonts: install all fonts, google-fonts-foobar: install foobar font only).

To be honest, I don't know whatever method is optimal.